src/Illuminate/Container/BoundMethod.php

```diff
@@ -157,16 +157,20 @@ protected static function getCallReflector($callback)
     protected static function addDependencyForCallParameter($container, $parameter,
                                                             array &$parameters, &$dependencies)
     {
-        if (array_key_exists($parameter->name, $parameters)) {
-            $dependencies[] = $parameters[$parameter->name];
+        if (array_key_exists($parameter->getName(), $parameters)) {
```

Comment:
Why was `->name` changed to `->getName()` everywhere? If that property no longer exists, that wasn't intentional and should be fixed on our side.

Reply:
That wasn't a fix, per say. It was just a move to use the documented methods for getting the name. :)